### ISSUE TITLE
Auto load "mf_user_keys.dic" from spiffs [idea]

### ISSUE THREAD

**Issue by ghiki** [RfidResearchGroup/proxmark3#2405]
**Is your feature request related to a problem? Please describe.**

When you do some attack, to load the keys file (whe you know, for testing for example) is very repetitive to do "-d xxx.dic" or "mem load -f xxxx" before doing

**Describe the solution you'd like**

Dictionary file (or files) in spiffs that loads when starting any type of attack  (obviously, this will do only when "ifdef WITH_FLASH")

**Describe alternatives you've considered**

Load manually before with

    mem load -f mfc_default_keys -m         -> upload MFC keys
    mem load -f t55xx_default_pwds -t       -> upload T55XX passwords
    mem load -f iclass_default_keys -i      -> upload iCLASS keys

or passing to the attack command manually

**Additional context**

**Comment by iceman1001** [comment]
you only need to upload the default dictionaries once on a RDV4 / spifffs based device.    Its stored in eeprom and read from eeprom when called in a function that takes advantage of that functionality.

**Comment by ghiki** [comment]
Yep, but for example, a "hf mf autopwn" or any other, doesn't use any dic file although this file exists in the spiffs, the main idea is to use if the device has flash memory and a file with user Keys exist on It (now, if you want to use It, you need to load previously with "data" command)

**Comment by iceman1001** [comment]
In that case are you thinking about updating `hf mf autopwn` to take advantage of the stored dictionary.

Uploading before to spiffs shouldn't affect autopwn to use it.  Like I said you only need to upload once. 
Well,  if the default dictionary file gets some additions you will of course re-upload it.   Doesn't happen that often.

**Comment by jgoggan** [comment]
I really like this idea and thought the same thing the first time I started working with the PM3 hardware.  On the previous hardware that I used, it automatically saved any previously cracked information so that if you ran a crack again on the same card, it would immediately complete because it already had the key handy.  For the PM3, it always seemed funny to me that if I ran "hf mf autopwn" and it took a while and then cracked it -- and then I immediately ran "hf mf autopwn" again, it would do the entire crack again.  I would love it if every time it was successfully at cracking a key, it stored it to a local mf_user_keys.dic that was auto-loaded for any new crack attempt.  I mean, once it has the key, use it next time!  :)

**Comment by iceman1001** [comment]
running attacks means running attacks,   if you store found keys which autopwn does,  you don't need to re-run attacks. 
You would use the already saved key file.

**Comment by jgoggan** [comment]
I think I over-simplified in my example.  In reality, I might have a stack of 10 or so cards that I am running through and doing attacks on -- and they all came from the same system.  Let's say that it finally cracks one of them after a fairly long attack.  That's great!  Now I have those keys.  Now I move onto the second card and start an attack.  Let's say it can't crack it, so I give up on that one.  I move onto card 3 -- and it cracks it, but none of the keys were the same as the first card.  Great -- got those keys too.

Now I move onto card 4.  And it runs forever and finally cracks it -- and I realize that most of the keys are the same as card #1!  They had reused a bunch of the keys.  If it kept a nice little cache of all cracked keys, it could have run through those first in a super-fast dictionary attack.

So that is what I'm referring to here.  I wish that there was a very easy way to go "take those keys you just found and store them in this key list or dictionary file so that you will try those first in future attacks."
